### DISCUSSION TITLE
Next album command / Plugin tutorial

### DISCUSSION THREAD

**Discussion by oliboss** [MiczFlor/RPi-Jukebox-RFID#2414] [Q&A]
I would like to have a rpc command "next_album" / "next_folder" to play the next album or folder. 

If I am not mistaken such a command currently doesn't exist. Is there a turorial or a "hello world" RPC command that I could use as a starting point for writing such a feature?

Thanks a lot in advance!

**Comment by devbar** [comment]
I'd like to push this topic up. I am on the same problem. I designed my box to switch between albums by a dedicated pair of buttons, but there is no rpc_command. I am really motivated to implement it.
